sql/core/src/test/resources/sql-tests/inputs/group-by-all.sql

```diff
@@ -75,8 +75,11 @@ select (id + id) / 2 + count(*) * 2 from data group by all;
 select country, (select count(*) from data) as cnt, count(id) as cnt_id from data group by all;
 
 -- correlated subquery should also work
-select (select count(*) from data d1 where d1.country = d2.country), count(id) from data d2 group by all;
+select country, (select count(*) from data d1 where d1.country = d2.country), count(id) from data d2 group by all;
 
--- make sure we report the right error when there's an attribute in correlated subquery
--- that is, to report UNRESOLVED_ALL_IN_GROUP_BY, rather than some random subquery error.
-select coutnry, (select count(*) from data d1 where d1.country = d2.country), count(id) from data d2 group by all;
```

Comment:
`coutnry` is an non-existing column and we should report normal missing column error.

Reply:
😢